pkg/cmd/server/start.go

```diff
@@ -155,9 +155,9 @@ func (o *PorchServerOptions) Config() (*apiserver.Config, error) {
 		return nil, fmt.Errorf("error creating self-signed certificates: %w", err)
 	}
 
-	if o.RecommendedOptions.Etcd != nil {
-		o.RecommendedOptions.Etcd.StorageConfig.Paging = utilfeature.DefaultFeatureGate.Enabled(features.APIListChunking)
-	}
+	// if o.RecommendedOptions.Etcd != nil {
+	// 	o.RecommendedOptions.Etcd.StorageConfig.Paging = utilfeature.DefaultFeatureGate.Enabled(features.APIListChunking)
```

Comment:
Couldn't you just remove this?

Reply:
Possibly but my knowledge of the Paging functionality in etcd is limited.
They removed it in the latest version. Plan was maybe to reintroduce it once I have the know how.
Can remove if it's cleaner. Don't think we are losing much.

Reply:
Not sure. Was hoping to reinstate once this PR was in. Paging was removed from the latest version so may be best to just delete.